packages/components/pie-button/test/component/pie-button.spec.ts

```diff
@@ -525,3 +570,110 @@ test.describe('Form Actions', () => {
         });
     });
 });
+
+test.describe('props', () => {
+    test.describe('isResponsive', () => {
+        test.describe('when not set', () => {
+            test('the button should not have the attribute', async ({ mount }) => {
+                const component = await mount(
+                    PieButton,
+                    {
+                        props,
+                        slots: {
+                            default: 'Click me!',
+                        },
+                    },
+                );
+
+                await expect(component.locator('button'))
+                    .not.toHaveAttribute('isResponsive', '');
+            });
+        });
+        test.describe('when set to true', () => {
+            test('the button should have the attribute', async ({ mount }) => {
+                const testProps:Partial<ButtonProps> = {
+                    ...props,
+                    size: 'xsmall',
+                    isResponsive: true,
+                };
+
+                const component = await mount(
+                    PieButton,
+                    {
+                        props: testProps,
+                        slots: {
+                            default: 'Click me!',
+                        },
+                    },
+                );
+
+                await expect(component.locator('button'))
+                    .toHaveAttribute('isResponsive', '');
```

Comment:
I think this should also work without the second parameter

Reply:
Ditto